docs/features/logforwarding/outputs/azure-monitor-log-forwarding.adoc

```diff
@@ -0,0 +1,174 @@
+=== Steps to forward to Azure Monitor Log via HTTP Data Collector API
+
+(https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell)
+
+. Create a secret containing your `shared_key` using the following command (primary or the secondary key for the workspace that's making the request):
++
+----
+ oc create secret generic azure-secret -n openshift-logging --from-literal=shared_key='<shared_key>'
+----
+
+. Create a Cluster Logging instance:
```

Comment:
I still see this section which we can remove regarding ClusterLogging

Reply:
got it